packages/spec/vaults/vaults.spec.ts

```diff
@@ -46,55 +46,154 @@ import {
 
 describe('Given the Aave Vaults', () => {
   describe('When an organization deploys a new vault', () => {
-    const organization = createNewWallet();
+    describe("And is single recipient of the vault's fees", () => {
+      const organization = createNewWallet();
 
-    beforeAll(async () => {
-      const setup = await fundErc20Address(
-        ETHEREUM_WETH_ADDRESS,
-        evmAddress(organization.account!.address),
-        bigDecimal('0.1'),
-      );
-      assertOk(setup);
+      beforeAll(async () => {
+        const setup = await fundErc20Address(
+          ETHEREUM_WETH_ADDRESS,
+          evmAddress(organization.account!.address),
+          bigDecimal('0.1'),
+        );
+        assertOk(setup);
+      });
+
+      it('Then it should be available in the organization vaults', async ({
+        annotate,
+      }) => {
+        annotate(`organization address: ${organization.account!.address}`);
+        const initialVault = await vaultDeploy(client, {
+          chainId: ETHEREUM_FORK_ID,
+          market: ETHEREUM_MARKET_ADDRESS,
+          deployer: evmAddress(organization.account!.address),
+          owner: evmAddress(organization.account!.address),
+          initialFee: bigDecimal('3'),
+          initialLockDeposit: bigDecimal('0.05'),
+          shareName: 'Aave WETH Vault Shares',
+          shareSymbol: 'avWETH',
+          underlyingToken: ETHEREUM_WETH_ADDRESS,
+        })
+          .andThen(sendWith(organization))
+          .andTee((tx) => annotate(`tx to deploy vault: ${tx.txHash}`))
+          .andThen(client.waitForTransaction)
+          .andThen((txHash) =>
+            vault(client, { by: { txHash }, chainId: ETHEREUM_FORK_ID }),
+          )
+          .map(nonNullable);
+        assertOk(initialVault);
+        annotate(`initial vault: ${initialVault.value?.address}`);
+
+        expect(initialVault.value.owner).toEqual(
+          organization.account!.address.toString(),
+        );
+        expect(initialVault.value.feeRecipients).toEqual([
+          expect.objectContaining({
+            address: organization.account!.address,
+            percent: expect.objectContaining({
+              formatted: '100',
+            }),
+          }),
+        ]);
+
+        const result = await vaults(client, {
+          criteria: {
+            ownedBy: [evmAddress(organization.account!.address)],
+          },
+        });
+
+        assertOk(result);
+
+        expect(result.value.items).toEqual([
+          expect.objectContaining({
+            owner: organization.account!.address,
+            address: initialVault.value!.address,
+          }),
+        ]);
```

Comment:
This part of the test is not needed, we have already tests fetching vault by `ownedBy` so I would remove this part

Reply:
What's the actual assertion here? Simply it does not fail? Can we make this more business oriented and see the revenue flowing to the addresses?

Reply:
I think you are already checking the owner in line `87` (creating vault and fetching vault by txHash and later assertion in the owner, that should be enough) so no need to fetch byOwner again.

Also it would be good to check when withdrawing fees that new accounts received the tokens.

Reply:
what the owner has to do with the multi-recipient aspect of this feature?

Reply:
I agree we should minimize duplication, but if the assertion indeed should be changed to something else (like I am suspecting) then effectively 1 of these 2 tests is not longer redundant with the existing one @juangm is referring to.